spark-plugins/src/main/java/co/cask/hydrator/plugin/spark/HashingTFFeatureGenerator.java

```diff
@@ -79,11 +81,12 @@ public void initialize(SparkExecutionPluginContext context) throws Exception {
       return context.getSparkContext().emptyRDD();
     }
     outputSchema = outputSchema != null ? outputSchema : config.getOutputSchema(input.first().getSchema());
-    final Map<String, String> mapping = config.getFeatureListMapping();
+    final Map<String, String> mapping = SparkUtils.getFeatureListMapping(config.outputColumnMapping);
```

Comment:
The getFeatureListMapping() is common for HashingTFFeatureGenerator and SkipGramFeatureGenerator and was being used in 4 different places.
Therefore, shifted the method to utils to avoid redundancy.
And regarding validation, we are now passing the map in the validation method directly line [#147](https://github.com/caskdata/hydrator-plugins/pull/492/files/0ade175af8e6ea6a61627332bb07cc93468881c2#diff-a4c33ebf066fe93c6a027acac168d929R147).